bigchaindb/core.py

```diff
@@ -136,11 +138,54 @@ def write_transaction(self, signed_transaction, durability='soft'):
         signed_transaction = deepcopy(signed_transaction)
         # update the transaction
         signed_transaction.update({'assignee': assignee})
+        signed_transaction.update({'assignment_timestamp': time()})
 
         # write to the backlog
         response = r.table('backlog').insert(signed_transaction, durability=durability).run(self.conn)
         return response
 
+    def reassign_transaction(self, transaction, durability='hard'):
```

Comment:
After thinking a bit more about this it would make sense for a single node to reassign the transaction back to itself. Lets say the node lost one of the notifications from the changefeed by reassigning the transaction to itself it will fire up a new changefeed notification.